src/core_plugins/kibana/public/home/components/sample_data_set_card.js

```diff
@@ -28,70 +28,40 @@ import {
   EuiToolTip,
 } from '@elastic/eui';
 
-import {
-  installSampleDataSet,
-  uninstallSampleDataSet
-} from '../sample_data_sets';
+const INSTALLED_STATUS = 'installed';
```

Comment:
I guess these are returned from the objects in `listSampleDataSets`?  Our folder structure of core plugins makes it difficult to push common consts in an intuitive place (the server side code of this is far away).   If we typescripted this and the files in `../sample_data_sets` we could at least show that the object being returned has a `status` and is one of `installed` or `not_installed`.  Suppose that's fine for this PR but I think it'd be really helpful to understand and follow this code if it was in typescript.